azkaban-web-server/src/main/java/cloudflow/docs/ExecutionDetailSample.md

```diff
@@ -0,0 +1,194 @@
+{
+  "executionId": "2",
+  "submitUser": "azkaban",
+  "submitTime": 1581383285656,
+  "concurrentOption": "skip",
+  "failureAction": "FINISH_CURRENTLY_RUNNING",
+  "experimentId": "defaultExperimentId",
+  "description": "defaultDescription",
+  "previousFlowExecutionId": "defaultPreviousExecutionId",
+  "notifyFailureFirst": false,
+  "notifyFailureLast": false,
```

Comment:
Can you use `notifyOnFirstFailure` and `notifyFailureOnExecutionComplete` as the names of these properties to maintain consistency with the rest of the endpoints?

Reply:
Will do.